redis_broker.go

```diff
@@ -64,15 +71,15 @@ func (cb *RedisCeleryBroker) SendCeleryMessage(message *CeleryMessage) error {
 func (cb *RedisCeleryBroker) GetCeleryMessage() (*CeleryMessage, error) {
 	conn := cb.Get()
 	defer conn.Close()
-	messageJSON, err := conn.Do("BLPOP", cb.queueName, "1")
+	messageJSON, err := conn.Do("BLPOP", cb.routes[defaultRoute].Queue, "1")
 	if err != nil {
 		return nil, err
 	}
 	if messageJSON == nil {
 		return nil, fmt.Errorf("null message received from redis")
 	}
 	messageList := messageJSON.([]interface{})
-	if string(messageList[0].([]byte)) != "celery" {
+	if string(messageList[0].([]byte)) != cb.routes[defaultRoute].Queue {
```

Comment:
in #113 I actually removed this test because if we BLPOP from a single queue we are assured that redis will give us back the same queue. Maybe we should check it is part of routes, or even that the "tasks" field of messageList[1] is actually the same as the taskName? Not sure yet, let me know what you think.

Reply:
I'm not sure if I totally understand, but I think I see the problem you're pointing at.  I agree - at this point it doesn't even confirm that the specified queue is being used, just that the whichever queue it's using behind the scenes is being both pushed to and popped from.  Confirming some additional features of the message would be a good idea, I'll take a look at that